runtime/parachains/src/paras.rs

```diff
@@ -990,44 +990,6 @@ impl<T: Config> Pallet<T> {
 	pub(crate) fn note_new_head_weight() -> Weight {
 		T::DbWeight::get().reads_writes(6, 6)
 	}
-
-	/// Fetches the validation code hash for the validation code to be used when validating a block
-	/// in the context of the given relay-chain height. A second block number parameter may be used
-	/// to tell the lookup to proceed as if an intermediate parablock has been with the given
-	/// relay-chain height as its context. This may return the hash for the past, current, or
-	/// (with certain choices of `assume_intermediate`) future code.
-	///
-	/// `assume_intermediate`, if provided, must be before `at`. This will return `None` if the validation
-	/// code has been pruned.
-	///
-	/// To get associated code see [`Self::validation_code_at`].
-	pub(crate) fn validation_code_hash_at(
```

Comment:
Removed since it is unused, correct?

Reply:
This got removed when I merge in master. Its not in the diff anymore